config/run-axe.js

```diff
@@ -0,0 +1,152 @@
+'use strict';
+
+const fs = require('fs');
+const urlParse = require('url').parse;
+const chromeLauncher = require('chrome-launcher');
+const CDP = require('chrome-remote-interface');
+const chalk = require('chalk');
+const runServer = require('./static-server');
+
+const REMOTE_CHROME_URL = process.env['REMOTE_CHROME_URL'];
+const AXE_JS = fs.readFileSync(__dirname + '/../node_modules/axe-core/axe.js');
+const PAGES = [
+  '/',
+  '/page-templates/landing/',
+  '/page-templates/docs/',
+];
+
+function launchChromeLocally(headless=true) {
+  return chromeLauncher.launch({
+    chromeFlags: [
+      '--window-size=412,732',
+      '--disable-gpu',
+      headless ? '--headless' : ''
+    ]
+  });
+}
+
+function getRemoteChrome() {
+  const info = urlParse(REMOTE_CHROME_URL);
+  if (info.protocol !== 'http:')
+    throw new Error(`Unsupported protocol: ${info.protocol}`);
+  return new Promise(resolve => {
+    resolve({
+      host: info.hostname,
+      port: info.port,
+      kill() { return Promise.resolve(); }
+    });
+  });
+}
+
+// This function is only here so it can be easily .toString()'d
+// and run in the context of a web page by Chrome. It will not
+// be run in the node context.
+function runAxe() {
+  return new Promise((resolve, reject) => {
+    window.axe.run((err, results) => {
+      if (err) return reject(err);
+      resolve(JSON.stringify(results.violations));
+    });
+  });
+}
+
+let getChrome = REMOTE_CHROME_URL ? getRemoteChrome : launchChromeLocally;
+
+Promise.all([runServer(), getChrome()]).then(([server, chrome]) => {
+  const chromeHost = chrome.host || 'localhost';
+  console.log(`Static file server is listening at ${server.url}.`);
+  console.log(`Chrome is debuggable on http://${chromeHost}:${chrome.port}.`);
+  console.log(`Running aXe on:`);
+
+  CDP({
+    host: chrome.host,
+    port: chrome.port,
+  }, client => {
+    const {Page, Network, Runtime} = client;
+    const pagesLeft = PAGES.slice();
+    const loadNextPage = () => {
+      if (pagesLeft.length === 0) {
+        console.log(`Finished visiting ${PAGES.length} pages with no errors.`);
+        terminate(0);
+      } else {
+        const page = pagesLeft.pop();
+        const url = `${server.url}${page}`;
+        process.stdout.write(`  ${page} `);
```

Comment:
Oh, I wanted it to look cool like all the fancy test runners!

![2017-07-12_15-43-34](https://user-images.githubusercontent.com/124687/28136780-fc177cc0-6718-11e7-8803-750b651de81f.png)